fedimint-server/src/consensus/engine.rs

```diff
@@ -472,6 +484,89 @@ impl ConsensusEngine {
             .expect("This is the only place where we write to this key");
     }
 
+    /// Creates the directory within the data directory for storing the database
+    /// checkpoints or deletes checkpoints before `current_session` -
+    /// `checkpoint_retention`.
+    fn initialize_checkpoint_directory(&self, current_session: u64) -> anyhow::Result<()> {
+        let checkpoint_dir = self.data_dir.join("db_checkpoints");
```

Comment:
Nit: would make a `fn db_checkpoints_dir() -> PathBuf` to avoid possibility of typos

Reply:
And maybe `fn db_checkpoint_dir(session_num: u64) -> PathBuf` maybe. Seems like it repeats couple of times.

Reply:
Done